Expensify/App
# Truncate Header Text if the chat participants more than two lines. Ad…

Truncate chat name header after two lines with an ellipsis.

### Fixed Issues
Fixes #1065 

### Tests
1. Create a chat room with more than three participants
2. Use chrome developer console to test the truncating with the various screen resolution
3. Add long chat message to make sure it doesn't affect the other layout
4. Open and close sidebar to make sure the sidebar work well

### Tested On


### Screenshots

#### Web & Mobile Web

![dTe385ZMuA](https://user-images.githubusercontent.com/11847279/103972512-bac42a00-519f-11eb-95bf-7893fbae1329.gif)

#### Desktop
![WFt4j2DgGn](https://user-images.githubusercontent.com/11847279/103972455-9ff1b580-519f-11eb-9bd4-56d3fc847085.gif)

#### iOS

![image](https://user-images.githubusercontent.com/11847279/103972388-7e90c980-519f-11eb-9ac7-ab545d1aea19.png)




#### Android

![image](https://user-images.githubusercontent.com/11847279/103972349-6620af00-519f-11eb-961b-4cf25f29ae22.png)
